### PR TITLE
Cadp 711: Alter Google Earth Engine Authentication method in geeutils package which is to be used in pipeline_utils package.

### PR DESCRIPTION
**Pull request by ankit94**
https://vulcan.atlassian.net/browse/CADP-711

### CODE REVIEW

File: geeutils/eu.py [hunk 1]
```diff
@@ -22,14 +22,15 @@
...
+def authenticate(allow_interactive=True) -> None:
     """Authenticate with the GEE API.
 
     A SERVICE_ACCOUNT_KEY environment variable must be set with the key to a
     Google Cloud Platform service account that has permissions in GEE.
 
     Args:
-        None
+        allow_interactive: If False, does not allow the interactive login workflow.
+                           If True, continue to interactive login workflow.
```
Comment (mattskone) [inline geeutils/eu.py]: Maybe add "(default True)" so users know

File: geeutils/eu.py [hunk 2]
```diff
@@ -39,6 +40,8 @@ def authenticate() -> None:
         service_account_key = os.environ['SERVICE_ACCOUNT_KEY']
     except KeyError:
         if not os.path.exists('/root/.config/earthengine/credentials'):
+            if allow_interactive == False:
```
Comment (mattskone) [inline geeutils/eu.py]: Not required, but more Pythonic would be 
```python
if not allow_interactive:
```

**Review by mattskone — APPROVED** [review]
Nice work!  A couple minor points, but this should work great.

File: geeutils/eu.py (hunk 1 above)
Comment (ankit94) [reply geeutils/eu.py]: taken care of.

**Pull request merged**